docs/sources/k6/next/javascript-api/jslib/aws/KinesisClient/_index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'KinesisClient'
+head_title: 'KinesisClient'
```

Comment:
```suggestion
```

We're not using this property in the Grafana docs, I think it's a leftover from the Gatsby docs.